src/Common/tests/System/PlatformDetection.cs

```diff
@@ -23,6 +23,9 @@ public static class PlatformDetection
             File.Exists(Path.Combine(Environment.GetEnvironmentVariable("windir"), "regedit.exe")));
         public static bool IsWindows10Version1607OrGreater { get; } = IsWindows &&
             GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 14393;
+        // Windows OneCoreUAP SKU doesn't have httpapi.dll
+        public static bool HasHttpApi { get; } = (IsWindows &&
```

Comment:
If on UAP all http.sys/httpapi.dll tests should be disabled (temporarily until we get to implement it properly on UAP).
The check doesn't hold when running these tests on a full Windows machine in UAP mode.